types/twilio-video/index.d.ts

```diff
@@ -279,13 +280,16 @@ export class RemoteTrackPublication extends TrackPublication {
     isTrackEnabled: boolean;
     kind: Track.Kind;
     track: RemoteTrack | null;
+    priority: TrackPriority;
 }
 export class RemoteTrackStats extends TrackStats {
     bytesReceived: number | null;
     packetsReceived: number | null;
 }
 export class RemoteVideoTrack extends VideoTrack {
     sid: Track.SID;
+    priority: TrackPriority;
```

Comment:
```suggestion
    priority: TrackPriority | null;
```

This is nullable. https://media.twiliocdn.com/sdk/js/video/releases/2.3.0/docs/RemoteVideoTrack.html